server/statistics/hot_peer_cache.go

```diff
@@ -294,19 +311,23 @@ func (f *hotPeerCache) getOldHotPeerStat(regionID, storeID uint64) *HotPeerStat
 }
 
 func (f *hotPeerCache) calcHotThresholds(storeID uint64) []float64 {
-	statKinds := f.kind.RegionStats()
-	ret := make([]float64, DimLen)
-	for dim, kind := range statKinds {
-		ret[dim] = MinHotThresholds[kind]
-	}
 	tn, ok := f.peersOfStore[storeID]
 	if !ok || tn.Len() < TopNN {
-		return ret
+		return f.defaultThresholds
 	}
-	for i := range ret {
-		ret[i] = math.Max(tn.GetTopNMin(i).(*HotPeerStat).GetLoad(i)*HotThresholdRatio, ret[i])
+	thresholds, ok := f.thresholdsOfStore[storeID]
+	if ok && time.Since(thresholds.updatedTime) <= DefaultThresholdsUpdateInterval {
```

Comment:
Will it affect write threshold?

Reply:
Yes, but less interval means update more frequently, so it will only make write cache more accurate